integration/tests/conftest.py

```diff
@@ -753,3 +753,8 @@ def precompiled_contract(web3_client, faucet, accounts):
         "precompiled/CommonCaller", "0.8.10", accounts[0]
     )
     return contract
+
+
+@pytest.fixture(scope="session")
+def default_cu_price(pytestconfig: Config) -> int:
+    return pytestconfig.environment.default_cu_price  # must be equal to compose.proxy.environment.DEFAULT_CU_PRICE
```

Comment:
please check that it can be empty for stands you did't add the value (like mainnet, tracer or docker_net)
it should fail on a start at least